webservices/common/models/dates.py

```diff
@@ -85,7 +85,7 @@ class ElectionClassDate(db.Model):
     senate_class = db.Column(db.Integer, index=True, doc=docs.SENATE_CLASS)
 
 
-class CalendarDate(BaseModel):
+class CalendarDate(db.Model):
```

Comment:
There is no more need of the `idx` column that the `BaseModel` provides?  I see the schema change below, so I'm assuming there's not; just wanted to double check!